sites/public/pages/applications/start/autofill.tsx

```diff
@@ -37,9 +36,8 @@ export default () => {
       setSubmitted(true)
       if (previousApplication && useDetails) {
         conductor.application = previousApplication
-      } else {
-        conductor.application = blankApplication()
```

Comment:
I had this here so if somebody went back in their browser history and changed to *not* autofilling the application, it'd be rewritten as a blank application. Otherwise they could never undo their change.

Reply:
@jaredcwhite , how close do you think we are to being able to merge the Next branch? If you're thinking next week, then can we get this in first? We're in a tough spot of not wanting to delay the Next updates any longer, but we also don't want to stack up a lot of dependencies.

Reply:
@seanmalbert Oh I want to get Next v10 ready to merge today, so I think this could follow suit.